src/course-page/CoursePage.jsx

```diff
@@ -35,7 +35,7 @@ function CoursePageConfigTile({
       <p>{coursePageStatus ? 'Enabled' : 'Disabled'}</p>
       <div>{description}</div>
       <div className={getEnabledButtonClasses()}>
```

Comment:
I don't see why we need to call an intermediary function here. Just add the entire `classNames` function call here. 
If that makes it harder to understand, you can just put it in a variable. So something like:

```jsx
const containerClasses = classNames('course-page-tile bg-white shadow p-3 m-d', {'disabled': !coursePageStatus});
... // Later
 <div className={containerClasses} ...
```